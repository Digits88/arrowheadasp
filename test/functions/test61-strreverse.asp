<!-- Test StrReverse function.
-->

StrReverse("12345") = <%= StrReverse("12345") %><br>
StrReverse("") = <%= StrReverse("") %><br>
