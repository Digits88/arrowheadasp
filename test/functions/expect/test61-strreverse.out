<!-- Test StrReverse function.
-->

StrReverse("12345") = 54321<br>
StrReverse("") = <br>
